### ISSUE TITLE
The training curves in different phrases

### ISSUE THREAD

**Issue by ChenX17** [mangushev/aligntts#1]
Hello,
When I train the first phrase using the align loss, at first the training loss is a positive number, then it becomes a negative number. Can you share the training curves in different phrases? Thank you!

**Comment by mangushev** [comment]
Hi, please see samples/curves folder now. It is negative since it is negated logarithm. We just training to the point when it becomes flat, does not decrease much any more. Also, phase 3 fine tuning mix density network

**Comment by ChenX17** [comment]
Thanks for your prompt reply^-^
It can be seen from the curve that you have trained 130k steps in phase 1, I also want to ask about the training time and the CPU you use.  Thanks!

**Comment by mangushev** [comment]
I used something like 16 cpu 16gb memory. I found that GPU does not do it faster when mix density is involved. As I mentioned learning rate needs to be 5e-5. I started with 1e-6 and it takes too long. So with 5e-5 maybe 40k steps will be enough. It took long to train. I use gcp spot instances. You may train for 1k steps and multiply to see how long it will take.

**Comment by mangushev** [comment]
I am closing since no further comments
